src/components/MakeElement.js

```diff
@@ -0,0 +1,15 @@
+const Create = (() => {
+  const element = (element) => {
+    const el = document.createElement(element);
+    return el;
+  };
+
+  const text = (text) => {
+    const txt = document.createTextNode(text);
+    return txt;
+  };
+
+  return { element, text };
+})();
+
+export default Create;
```

Comment:
Nice done using IIFE and the revealing module pattern.

Since you are using Webpack to modularize and bundle your code, you could simplify this and let Webpack take care of the modules/scope:

```js
const element = (element) => document.createElement(element);
const text = (text) => document.createTextNode(text);

export default { element, text };
```

Take a look at `dist/main.js`, to see what webpack is doing for you: Modules get wrapped into functions isolating their scope.

> Each module is a piece of code that is executed once it is loaded. In that code, there may be declarations (variable declarations, function declarations, etc.). **By default, these declarations stay local to the module.**
You can mark some of them as exports, then other modules can import them. 

https://exploringjs.com/es6/ch_modules.html#_browsers-scripts-versus-modules
https://medium.com/webpack/brief-introduction-to-scope-hoisting-in-webpack-8435084c171f

Reply:
You're right. I've implemented that